modules/control-plane-machine-set-operator.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * operators/operator-reference.adoc
+
+[id="control-plane-machine-set-operator_{context}"]
+= Control Plane Machine Set Operator
+
+[NOTE]
+====
+This Operator is available for Amazon Web Services (AWS), Microsoft Azure, and VMware vSphere.
+====
+
+[discrete]
+== Purpose
+
+The Control Plane Machine Set Operator automates the management of control plane machine resources within an {product-title} cluster.
+
+[discrete]
+== Project
+
+link:https://github.com/openshift/cluster-control-plane-machine-set-operator[cluster-control-plane-machine-set-operator]
+
+[discrete]
+== CRDs
+
+* `controlplanemachineset.machine.openshift.io`
+** Scope: Namespaced
+** CR: `ControlPlaneMachineSet`
+** Validation: No
```

Comment:
What would this normally contain?

Reply:
This is what most of them look like. A lot of Operators only have one CRD lsited, a few have several (CAPI, for example). Some don't have a CRDs section at all. I am not really sure why that is.

For this one, I was not sure about `Scope` (I see `Namespaced` and `Cluster` in other listings) or `Validation`.

Reply:
Namespaced scope is correct, validations, well we do apply various validations, but I'm not sure how much of that needs to be documented here

Reply:
Scanning the otehrs, I don't see explanation of validations, so I think it is ok to just flip it over to `Yes` - will do that now